utils/providers.py

```diff
@@ -368,7 +368,7 @@ def clear_provider_by_type(prov_class, validate=True):
         logger.info(' Providers exist, so removing all {} providers'.format(prov_class))
         paginator.results_per_page('100')
         sel.click(paginator.check_all())
-        toolbar.select('Configuration', 'Remove {} Providers from the VMDB'.format(string_name),
+        toolbar.select('Configuration', 'Remove {} Providers'.format(string_name),
```

Comment:
Yes, but this change may not have hit downstream yet, we need to ensure it's been merged :)